### PR TITLE
Add Ziggy library

### PR DESCRIPTION
**Pull request by osbre**
[Ziggy](https://github.com/tighten/ziggy) is a popular library that now comes included in new Inertia.js + Laravel projects by default.

Detected if:
- ~has global `window.route` function.~
- has `Ziggy` variable in the top-level scope (but not global scope, accessible via `Ziggy`, but not `window.Ziggy`)
- mentions `window.Ziggy` in the source code

I included `window.Ziggy` into the `source` section because there are cases when `Ziggy` is not explicitly defined but still mentioned in the source code, which allows the detection. Such case can be tested on this [this web page](https://app.sitesauce.app/login).

Exampels for testing:
- https://gistlog.co/
- https://useflipp.com/register
- https://app.sitesauce.app/login
- https://app.blogcast.host/register
- https://app.notionmailer.com/login
- https://app.formbold.com/register

### CODE REVIEW

File: src/technologies/z.json [hunk 1]
```diff
@@ -326,6 +326,25 @@
...
+    ],
+    "description": "Ziggy is a library that allows using Laravel named routes in JavaScript",
+    "icon": "Ziggy.png",
+    "implies": [
+      "Laravel",
+      "Inertia.js"
+    ],
+    "js": {
+      "route": ""
+    },
```
Comment (nurbek91) [inline src/technologies/z.json:341]: I'm not sure if this can be done.

File: src/technologies/z.json [hunk 2]
```diff
@@ -326,6 +326,25 @@
...
+    "script": [
+      "window.Ziggy"
+    ],
```
Comment (nurbek91) [inline src/technologies/z.json:344]: image
no such attribute exists.

File: src/technologies/z.json [hunk 3]
```diff
@@ -326,6 +326,25 @@
     "saas": true,
     "website": "https://zid.sa"
   },
+  "Ziggy": {
+    "cats": [
+      59
+    ],
+    "description": "Ziggy is a library that allows using Laravel named routes in JavaScript",
```
Comment (nurbek91) [inline src/technologies/z.json:333]: `"description": "Ziggy is a library that allows using Laravel named routes in JavaScript.",`

File: src/technologies/z.json (hunk 2 above)
Comment (nurbek91) [reply src/technologies/z.json:344]: Can you share examples for testing?

**Review by nurbek91 — COMMENTED** [review]
Hey mate, thanks for PR!!!

File: src/technologies/z.json (hunk 2 above)
Comment (osbre) [reply src/technologies/z.json]: Thanks, fixed the attribute name. 

Regarding examples:

Some websites have it on every page,
- https://gistlog.co/
- https://useflipp.com/register
- https://app.sitesauce.app/login
- https://app.blogcast.host/register
- https://app.notionmailer.com/login

While others only include it for authenticated users:
- https://www.larabug.com/
- https://ploi.io/

File: src/technologies/z.json (hunk 1 above)
Comment (osbre) [reply src/technologies/z.json]: Ok, I've updated the file to remove this bit

File: src/technologies/z.json [hunk 4]
```diff
@@ -326,6 +326,25 @@
...
+    "implies": [
+      "Laravel",
+      "Inertia.js"
+    ],
+    "js": {
+      "Ziggy": ""
+    },
+    "scripts": [
+      "window.Ziggy"
+    ],
```
Comment (nurbek91) [inline src/technologies/z.json:344]: `"scripts": "window\\.Ziggy",`

**Review by nurbek91 — APPROVED** [review]
Thanks mate, significant PR, there were a lot of Laravel projects that were not tracked, now thanks to you part is fixed

**Review by nurbek91 — APPROVED** [review]

**Pull request merged**